Tests/MessagingInApp/MessagingInAppImplementationTest.swift

```diff
@@ -165,11 +147,6 @@ class MessagingInAppImplementationTest: UnitTest {
         let givenAction = "gist://close"
         let givenName = String.random
 
-        backgroundQueueMock.addTrackInAppDeliveryTaskReturnValue = (
-            success: true,
-            queueStatus: QueueStatus.successAddingSingleTask
-        )
-
```

Comment:
[Reason](https://github.com/customerio/customerio-ios/pull/461/files#r1441777027)

Reply:
instead of removing, we need to test/comment to check if this is being sent to EventBus mock